### PR TITLE
Use archunit freezing rules to accept violations

### PR DESCRIPTION
**Pull request by wolfs**
instead of our home-grown solution.

### CODE REVIEW

**Comment by wolfs** [comment]
@bot-gradle test QFL

File: subprojects/distributions-dependencies/build.gradle.kts [hunk 1]
```diff
@@ -12,7 +12,7 @@ plugins {
 }
 
 val antVersion = "1.10.9"
-val archunitVersion = "0.16.0"
+val archunitVersion = "0.19.0"
```
Comment (blindpirate) [inline subprojects/distributions-dependencies/build.gradle.kts:15]: `0.20.0` was released yesterday.

**Review by blindpirate — APPROVED** [review]
I spent some time on understanding the whole architecture-test project and it looks good to me. If with latest `0.20.0` all green I think it's good to go.

File: subprojects/distributions-dependencies/build.gradle.kts (hunk 1 above)
Comment (wolfs) [reply subprojects/distributions-dependencies/build.gradle.kts:15]: I switched to 0.20.0.

**Comment by wolfs** [comment]
@bot-gradle test and merge

**Comment by wolfs** [comment]
@bot-gradle test and merge

**Comment by wolfs** [comment]
@bot-gradle test and merge

**Pull request merged**